core/src/main/java/com/homeaway/streamingplatform/configuration/SchemaRegistrarConfig.java

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2018 Expedia Group.
+ * All rights reserved.  http://www.homeaway.com
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.homeaway.streamingplatform.configuration;
+
+import java.util.Map;
+
+import javax.validation.Valid;
+
+import lombok.Data;
+import lombok.NonNull;
+
+@Data
+public class SchemaRegistrarConfig {
+
+    @Valid
+    @NonNull
+    String className;
+
+    Map<String, ?> properties;
```

Comment:
why `?` type parameter?
Won't this be a `String` or `Object`?
If you make it an `Object` doesn't that simplify the need to copy the map in the `loadSchemaRegistrar` above?